internal/server/v1/info/golden/variable_group_info/demographics.json

```diff
@@ -117,20 +108,14 @@
         "id": "dc/g/Person_Age",
         "specializedEntity": "Age",
         "displayName": "Person by Age",
-        "descendentStatVarCount": 27197
+        "descendentStatVarCount": 27225
       },
       {
         "id": "dc/g/Person_AgeGroupClassification",
         "specializedEntity": "Age Group Classification",
         "displayName": "Person by Age Group Classification",
         "descendentStatVarCount": 2
       },
-      {
-        "id": "dc/g/Household_BenefitsStatus",
```

Comment:
I assume the dc/g/Household* deletions are due to the vertical_spec changes for hourshold.